src/js/jsx/sections/libraries/LibraryList.jsx

```diff
@@ -39,28 +77,47 @@ define(function (require, exports, module) {
          * @private
          * @param {string} libraryID Selected item ID
          */
-        _handleChange: function (libraryID) {
-            this.props.onLibraryChange(libraryID);
+        _handleChangeLibrary: function (libraryID) {
+            if (_LIBRARY_COMMANDS.indexOf(libraryID) === -1) {
+                this.props.onLibraryChange(libraryID);
+                return;
+            }
+
+            var selectedCommand = libraryID,
+                newLibraryName = selectedCommand === _RENAME_LIBRARY ? this.props.selected.name : "";
+
+            this.setState({
+                command: selectedCommand,
+                newLibraryName: newLibraryName
+            });
+
+            // Cancel Datalist selection
+            return false;
```

Comment:
The bottom three items in the libraries menu are commands and should not be selected by the user. Returning false will indicate Datalist to not set these items as selected. I added the callback comment here https://github.com/adobe-photoshop/spaces-design/pull/2125/files#diff-2679935a0e56611617e48e9185145f97R46